src/exceptions/InvalidIDException.ts

```diff
@@ -0,0 +1,9 @@
+import HttpException from './HttpException';
+
+class InvalidIDException extends HttpException {
+    constructor(id: string) {
+        super(400, `id ${id} is invalid`);
```

Comment:
Consider using something like [http-status-codes](https://www.npmjs.com/package/http-status-codes) to unify statuses throughout the app. files